lib/tinycolor.dart

```diff
@@ -13,18 +12,15 @@ export 'hsl_color.dart';
 export 'color_extension.dart';
 
 class TinyColor {
-  Color originalColor;
+  final Color originalColor;
   Color _color;
 
-  TinyColor(Color color) {
-    this.originalColor =
-        Color.fromARGB(color.alpha, color.red, color.green, color.blue);
-    this._color =
-        Color.fromARGB(color.alpha, color.red, color.green, color.blue);
-  }
+  TinyColor(Color color)
+      : this.originalColor = color,
```

Comment:
I believe assigning directly is just passing the reference? original code wants to create an identical `Color`.

Reply:
actually, color is immutable, you can even instantiate it with const, but when I forked it, I believe the creator of the package thought it was mutable, because the method setOpacity was not even implemented correctly.